rclcpp/include/rclcpp/node.hpp

```diff
@@ -118,7 +118,7 @@ class Node
   /* Create and return a Publisher. */
   template<typename MessageT>
   rclcpp::publisher::Publisher::SharedPtr
-  create_publisher(const std::string & topic_name, size_t queue_size);
+  create_publisher(const std::string & topic_name, rmw_qos_policy_t qos_policy);
```

Comment:
Since `rmw_qos_policy_t` is not a primitive type should it be passed as a const reference?

Same in all other functions signatures where it is passed.